docs/src/Components.jsx

```diff
@@ -225,6 +225,20 @@ var Components = React.createClass({
             <small>We currently use a small sample of <a href="http://www.alessioatzeni.com/meteocons/" title="Meteocons">Meteocons</a></small>
           </article>
 
+          <article id="stepper">
+            <h3>Stepper</h3>
+            <p>Display an input with a number value that is surrounded by decrement and increment buttons</p>
+            <CustomComponent codeText={fs.readFileSync(__dirname + '/../examples/Stepper.jsx', 'utf8')} />
+            <h4>Attributes</h4>
+            <ul>
+              <li><code>value</code> Number - The value of the stepper</li>
```

Comment:
I think this is ok, the onChange call will re-render the stepper passing in a new actual value...